azure-devops/azext_devops/dev/team/wiki.py

```diff
@@ -0,0 +1,220 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import webbrowser
+from knack.util import CLIError
+from azext_devops.dev.common.services import (get_wiki_client,
+                                              get_core_client,
+                                              get_git_client,
+                                              resolve_instance,
+                                              resolve_instance_and_project,
+                                              resolve_instance_project_and_repo)
+
+
+_DEFAULT_PAGE_ADD_MESSAGE = 'Added a new page using Azure DevOps CLI'
+_DEFAULT_PAGE_UPDATE_MESSAGE = 'Updated the page using Azure DevOps CLI'
+_DEFAULT_PAGE_DELETE_MESSAGE = 'Deleted the page using Azure DevOps CLI'
+
+
+def create_wiki(name, wiki_type='projectwiki', mapped_path=None, version=None,
+                organization=None, project=None, repository=None, detect=None):
+    """Create a wiki.
+    :param name: Name of the new wiki.
+    :type name: str
+    :param wiki_type: Type of wiki to create.
+    :type wiki_type: str
+    :param version: Repository branch name to publish the code wiki from. Only required for codewiki type.
+    :type version: str
+    :param mapped_path: Mapped path of the new wiki e.g. '/' to publish from root of repository.
+    Not required for project wiki.
```

Comment:
lets have common thing across `Only required for codewiki type.` or `Not required for project wiki.`